endpoints.php

```diff
@@ -1743,8 +1764,14 @@ function perform_batched_upload_to_kaltura($videos, $use_kaltura_player)
 
                 if ($response->status === EntryStatus::READY) {
                     $node = save_video_upload_status($video, $videos_to_upload, UploadStatus::COMPLETE);
-                    TapestryHelpers::saveAndDeleteLocalVideo($node, $response, $use_kaltura_player, $video->file->file_path);
+                    TapestryHelpers::saveAndDeleteLocalVideo($node, $kalturaApi, $response, $use_kaltura_player, $video->file->file_path);
```

Comment:
The kalturaApi parameter does not seem to be used in the implementation of TapestryHelpers::saveAndDeleteLocalVideo. Could you explain why this parameter was added?

Reply:
Nice catch. This was a leftover piece of code and that argument shouldn't be there. I have just removed it.